<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Пиданов Сергей Александрович</author>
 <class>frmSotr</class>
 <widget class="QWidget" name="frmSotr">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>453</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Сотрудники</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QTableView" name="tableSotr">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>56</y>
     <width>445</width>
     <height>149</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboVidUslugi">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>6</y>
     <width>193</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Все...</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>9</y>
     <width>103</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Вид услуги:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>212</y>
     <width>397</width>
     <height>229</height>
    </rect>
   </property>
   <property name="title">
    <string>Общая информация</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>23</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>ФИО:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sotrFIO">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>20</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>77</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Телефон:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sotr_nom">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>76</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>9-999-999-99-99; </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>49</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Адрес:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sotrAddres">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>48</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="sotrPol">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>76</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>М</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ж</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>77</y>
      <width>29</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Пол:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>105</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Должность:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sotrDoljn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>104</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="sotrInfo">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>132</y>
      <width>301</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>134</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Доп. инфо:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ApplyBut">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>200</y>
      <width>103</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить...</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/res/icons/6.ico</normaloff>:/res/icons/6.ico</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>32</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QToolButton" name="add_sotr">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>3</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/res/icons/7.ico</normaloff>:/res/icons/7.ico</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="del_sotr">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>3</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/res/icons/5.ico</normaloff>:/res/icons/5.ico</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="edit_sotr">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>3</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/res/icons/11.ico</normaloff>:/res/icons/11.ico</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>4</number>
  </property>
  <property name="gridDeltaY">
   <number>4</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
